src/GoogleSigninButton.tsx

```diff
@@ -13,18 +13,18 @@ interface RNGoogleSignStaticsType {
 }
 const RNGoogleSignin: RNGoogleSignStaticsType = NativeModules.RNGoogleSignin;
 
-export const GoogleSigninButton = ({ onPress, ...props }: GoogleSigninButtonProps) => {
+export const GoogleSigninButton = (props: GoogleSigninButtonProps) => {
```

Comment:
On press was not being gathered in `props` which is later spread to `RNGoogleSigninButton` on line 43.

Reply:
hello, thanks for the fix, can you please fix the CI? 👍